engine/baml-lib/jinja/src/evaluate_type/types.rs

```diff
@@ -304,7 +304,7 @@ impl PredefinedTypes {
         };
 
         // Any vars that are in both branches are merged
-        // Any vars that are only in one branch, unioned with undefined
+        // Any vars that are only in one branch, united with undefined
```

Comment:
```suggestion
        // Any vars that are only in one branch, unioned with undefined
```

"unioned" is technically an invalid word but is deliberately used here to reference type unions